<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCalendarWidget" name="data">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>401</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="naglowek">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kalkulator obciążenia treningowego</string>
    </property>
   </widget>
   <widget class="QLabel" name="DataLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>119</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Podaj datę treningu</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Dane">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>391</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Dane do klakulatora</string>
    </property>
    <widget class="QLabel" name="Kg_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>53</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>  kg</string>
     </property>
    </widget>
    <widget class="QLabel" name="KategoriaLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Podaj kategorię ćwiczenia</string>
     </property>
    </widget>
    <widget class="QLabel" name="PowtLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Podaj ilość powtórzeń</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PrzejdzDo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>351</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Przejdź do podsumowania</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dodaj">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>351</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dodaj ćwiczenie</string>
     </property>
    </widget>
    <widget class="QLabel" name="MasaLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Podaj swoją masę ciała</string>
     </property>
    </widget>
    <widget class="QLabel" name="CwiczenieLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Podaj nazwę ćwiczenia</string>
     </property>
    </widget>
    <widget class="QLabel" name="Kg">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>240</y>
       <width>53</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>  kg</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cwiczenie">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="masa">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="kategoria">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>Ćwiczenie siłowe</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Ćwiczenie siłowe</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Olimp. podn. ciężarów</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hipertroficzne</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Plajometryczne</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Akcesoryjne</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rehabilitacyjne</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="powtorzenia">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>200</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="serie">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="CiezarLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Podaj ciężar roboczy*</string>
     </property>
    </widget>
    <widget class="QLabel" name="Adnotacja2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>  masę ciała + ew. dodatkowe obciążenie</string>
     </property>
    </widget>
    <widget class="QLabel" name="SerieLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Podaj ilość serii</string>
     </property>
    </widget>
    <widget class="QLabel" name="Adnotacja">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>*w przypadku ćwiczenia z własną masą ciała podaj swoją</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ciezar">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>240</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Wyniki">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>831</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Wyniki</string>
    </property>
    <widget class="QPushButton" name="Wyczysc">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>210</y>
       <width>311</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Wyczyść wszystko</string>
     </property>
    </widget>
    <widget class="QLabel" name="P1">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>50</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Łączny cieżar podczas treningu:</string>
     </property>
    </widget>
    <widget class="QLabel" name="P2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>90</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ciezar jako wielokrotność masy ciała:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Kg_3">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>50</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>  kg</string>
     </property>
    </widget>
    <widget class="QLabel" name="Podsumowanie">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Podsumowanie -</string>
     </property>
    </widget>
    <widget class="QLabel" name="P3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>130</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Status treningu:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="ListaCiezarow">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>61</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="DataPodsumowania">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="PodsStatus">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>130</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="PodsWielokrotnosc">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>90</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="zalecenie">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>170</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QListWidget" name="ListaCwiczen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>241</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="ListaVol">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>71</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="PodsCiezar">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>50</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
